### DISCUSSION TITLE
Remote Desktop Services does not work after hardening

### DISCUSSION THREAD

**Discussion by SingingOwl** [0x6d69636b/windows_hardening#157] [Q&A]
I'm trying to harden a RDS (Remote Desktop Services) setup so that it achieves as high a score on a CIS Benchmark for Windows Server 2016 1.4.0 compliance scan (Nessus). 

However, each time I do a HailMary using the finding lists for CIS Benchmark for Windows Server 2016 (1.3.0 and 2.0.0) on the RDS machine, RDS ends up not working. It is unable to communicate with Windows Internal Database. I am unable to load my RDS setup and see nothing in Server Manager. It just says it is trying to load connection broker. 

Server setup:

1x AD server (single DC domain) - 1x RDS server

I am wonder if I need to isolate which CIS item is causing the issue.

Before hardening screenshots of RDS (server manager)
![image](https://github.com/0x6d69636b/windows_hardening/assets/48266053/e799d0a0-e79a-481a-b787-1ac67e15748b)
![image](https://github.com/0x6d69636b/windows_hardening/assets/48266053/669504c9-431e-4a64-954c-afe0abf97e9a)

After hardening screenshots of RDS (server manager)
![image](https://github.com/0x6d69636b/windows_hardening/assets/48266053/ce0ed26e-4a33-4538-b867-495dadceeaa5)
![image](https://github.com/0x6d69636b/windows_hardening/assets/48266053/c19c4921-6658-4239-bd00-5318e7d19d85)

**Comment by 0x6d69636b** [comment]
You may find this https://github.com/scipag/HardeningKitty/issues/35 or this https://github.com/scipag/HardeningKitty/issues/37 helpful. Otherwise I suggest you use the CIS Benchmark documentation, they document the impact of all their recommendations.

**Reply by SingingOwl** [reply]
Thank you for your reply. I have tried the solution in https://github.com/scipag/HardeningKitty/issues/37 to no avail. 

I hadn't checked out #35 before. Let me see how it matches with CIS items. 

I have gone through the CIS benchmark guide, however, there isn't very obviously related items other than items such as those mentioned in #37. 

I did notice that after I did a full hardening on the connected AD server, things had stopped working. I have tried restoring that DC but the RDS issue still remains. I will be doing a full reinstallation of this test environment and trying hardening only on the RDS machine first. And trying to isolate which items might be causing the issue.
